### DISCUSSION TITLE
Sem retorno nos processos seletivos e/ou no cadastro de curriculo

### DISCUSSION THREAD

**Discussion by dacferreira** [frontendbr/forum#493] [Discussão]
Fala galera, na paz ?

# O que está acontecendo

Ultimamente eu tenho colocado curriculo em algumas empresas que divulgam vagas em algumas redes sociais (Linkedin, Slack...). Porém, sem retorno.

Já cansei de participar de processos seletivos onde o entrevistador me diz que vai dar retorno e nunca dá. Eu só queria saber por que muitas empresas/profissionais agem dessa forma e não dão retorno para os candidatos.

Pelo menos comigo é um angustia e ansiedade que acontece quando envio o currículo para uma empresa tendo ela uma vaga no seu perfil. “Será que eles receberam?” “Será que já leram?” “Será que tem alguma vaga para mim?” “Será que me chamarão para o processo seletivo?” Será?… Será?… Será?…

Será que as empresas/profissionais não estão tendo consideração pelos candidatos e por isso acabam não se importando em dar retorno?!

**Comment by damasio34** [comment]
Eu sinceramente acho que seja pouco caso das empresas/entrevistadores, por receberem muitos perfis/currículos acabam negligenciando quanto ao feedback aos candidatos. Um vez li no slack um 'entrevistador' reclamando de um candidato que não deu feedback, uns se uniram à reclamação e outros disseram:
- Ah, é isso que vocês fazem com a gente, está aí o troco, sinta o gostinho, rs.
Eu apenas ri.

O "ideal" é que um equilíbrio, ao menos uma resposta automática, um e-mail padrão informando ao candidato que ele não foi selecionado ou que não se enquadra no perfil, um pouquinho de consideração e respeito com o candidato.

**Comment by bernardodiasc** [comment]
Se a empresa ignora e nunca retorna, acho que é pura falta de respeito. Mas sei que em muitos casos a fila de avaliação é longa e o processo pode demorar mesmo.

**Comment by RatoX** [comment]
Daqui a pouco vai rolar um **awesome-empresas-que-dao-feedback**

**Comment by MarcoWorms** [comment]
Olha cara, na minha opinião isso tem muito a ver com a cultura ruim das empresas. Aqui na @Pagarme a gente sempre é direto ao ponto e quando o candidato ja passou pela triagem de currículos nós respondemos as pessoas bem rápido (até 2 dias) depois de cada fase. Posso dizer isso baseado em fatos pois eu participo do processo seletivo aqui e a gente toma muito cuidado pra não deixar pessoas esperando e nem perder horarios. A única fase que pode demorar um pouco é entre você enviar o currículo e a gente fazer o primeiro contato, mas isso acontece por causa da disponibilidade das vagas e momento da contratação.

O processo seletivo aqui é muito legal pois os desenvolvedores estão envolvidos em 100% do processo. A gente não faz whiteboard-challenges e nem nada capcioso, o que vale é a capacidade da pessoa de lidar com os problemas do mundo real. Quando reprovamos um candidato aqui nós tentamos dar um feedback completo de tudo que vimos que acarretou na recusa e até recomendamos material de estudo para retentar o processo após alguns meses.

**Comment by leandroruel** [comment]
uma vez a muito tempo quando participei de um processo seletivo de uma empresa, apos uns dias eles me mandaram um email falando que gostaram do meu perfil, mas que nao se encaixava na vaga, mas que minha trajetória era inspiradora e me desejavam boa sorte. achei tão legal que nem fiquei triste.

**Comment by alanhr** [comment]
Eu também acho uma falta de respeito da empresa que não da o feedback para o candidato. O bom de quando você recebe um é que você pode correr atras dos seus pontos fracos. 
Na empresa onde eu trabalho sempre damos o Feedback para os candidatos, porque existe aquele velho ditado : "Não faça com os outros o que você não gostaria que fizessem com você".

**Comment by dacferreira** [comment]
@MarcoWorms muito bom saber que tem empresas assim. 
Feedback é sempre bem vindo, pois é uma das melhores formas de medição de desempenho e ajuda a criar confiança, visar a solução de problemas e o aprimoramento das habilidades dos profissionais.

**Comment by lucasjs** [comment]
Que discussão maravilhosa :heart: finalmente!!!

Ainda bem que estou trabalhando, porque eu estava cansado de ficar esperando resposta de algumas empresas. O pior é você participar de várias fases, gastar seu tempo com testes e uma pessoa não tem alguns minutos só para mandar um "espere mais um pouco", "vamos contratar só daqui uns meses", "você não passou", etc. Fica mais complicado ainda quando você está saindo de uma empresa e/ou fazendo faculdade.  Já houve casos que fui muito bem, participei de várias fases, peguei chuva, trânsito, cancelei consulta, cheguei atrasado na faculdade, para não receber nem um e-mail, uma ligação, sei lá. Acabamos ficando ansiosos, sem saber o que fazer, se corremos atrás, se esperamos. Isso é **PÉSSIMO**.

Eu fico muito feliz quando uma empresa dá um retorno, inclusive hoje sou amigo de pessoas que me disseram "não" na entrevista, de tão legais que foram comigo e quis manter contato. Se você trata os candidatos com respeito, sua empresa se torna respeitada, admirada, e as pessoas mesmo recebendo o não, um dia podem voltar a participar dos processos seletivos, indicar amigos. Sem contar que as pessoas se sentem muito bem com um feedback de entrevista. É uma sensação muito boa. As empresas só tem a ganhar dando um retorno.

Parece que as pessoas esquecem que há humanos se candidatando, que possuem vida, que é importante um bom tratamento. Empresa que faz isso só se queima, pois uma pessoa vai contando para outra e isso se espalha. É uma falta de respeito **ABSURDA**. Mas se esquecem que o mundo dá voltas..

**Comment by rubenmarcus** [comment]
Po vou falar um caso curioso pra vocês:

Em Abril de 2014 eu estava desempregado depois de um tempão fazendo freelas, e trabalhando em startups pequenas, eu decidi ir pra uma empresa maior.

Ai vi uma vaga no trampos pra Centauro e me candidatei. A consultoria que estava fazendo a seleção (ATMA) , me respondeu um dia depois, enviando um teste para eu fazer ( um mockup ) tinha que torna-lo responsivo. Beleza fiz em 2h, mandei, em 1 semana tive o feedback. Queriam uma entrevista presencial.

Nesse meio tempo de 1 semana, me chamaram por uma consultoria Indiana que ficava na alameda santos para a  mesma vaga. Disys acho o nome. Beleza. Escritório padrão entrevista com recrutador padrão , "fazer uma redação, falar sobre sua vida profissional e pessoal, no papel e na entrevista", deixar contatos de ex chefes. beleza.

Bom..
2 semanas depois, eu estava na Centauro sendo entrevistado pelo time de TI, e me apresentaram a proposta de trabalho, na terceira semana eu fechei contrato e já ia começar.

a Disys nunca me deu feedback. 1 mes e meio depois recebi o e-mail: Ruben sentimos muito, mas a vaga que você concorria foi preenchida já.

Os caras nem sabiam que por mim mesmo.

Por isso, que muitas vezes a "culpa" não é do candidato, mas muitas vezes de quem faz o processo de seleção.

**Comment by kvnol** [comment]
Muitas empresas, podemos ver nos requisitos, não possuem um gestor do time de desenvolvimento, um cara sênior que faz as entrevistas. 

O que me parece é que a pessoa do RH chega nos desenvolvedores e perguntam "vamos contratar um cara, o que vocês fazem aí?", monta uma lista de requisitos imensa que muitas vezes não faz sentido, quase fala que horário de almoço é benefício e outras baboseiras. 

Enfim, já fiquei sem respostas também para diversas empresas mas galera, pode ter certeza que se a empresa não mandou nem um "recebemos seu currículo" ou "infelizmente você não foi aprovado", nós (desenvolvedores) que saímos no lucro. Uma empresa assim não mando meu currículo nunca mais.

**Comment by ghost** [comment]
Opa achei uma brecha para participar do forum :)

Nunca trabalhei na área só estudo por enquanto, certo dia vi uma vaga de jr no frontendbr vagas, e me candidatei o cara pediu para eu mandar alguma coisa pra ele ver, eu enviei o link do github e um projetinho da apostila da caelum, mas não tive retorno eu imaginei que o e-mail com os links tinham ido para a caixa de span depois de alguns meses a vaga continuava aberta e eu entrei em contato de novo trocamos alguns emails mais depois sumiu ai percebi que foi falta de bom senso do cara, fiquei chateado pacas achei que ia ter um norte com o feedback mas não rolou, a vaga ainda está aberta lá quem querer se arriscar :)

**Comment by herberthenrique** [comment]
> Opa achei uma brecha para participar do forum :)

Sempre participe @Bruno413, faz bem :) 

Sobre feedback: 
Eu entendo não haver um feedback (não concordo) caso você não passe na fase de triagem de currículos, pois dependendo do lugar o recrutador acaba recebendo milhares de emails, cvs, etc.
Caso você passe dessa "primeira fase" o mínimo que você deveria receber seria um e-mail agradecendo ter participado do processo.

Agora, eu realmente fico feliz quando há um feedback real sobre as razões de não ter passado, especialmente se foram por motivos técnicos. Eu já recebi feedbacks e vi muitas empresas dando feedback de "você não passou pq faltou conhecimento em x, y e z". 

Acho que o pessoal de tech, deveria sempre pensar em poder dar esse tipo de feedback quando estamos contratando.

**Comment by pedrobritto** [comment]
Isso já aconteceu comigo. Enviei meu currículo (bem modesto, sou bem iniciante ainda) para 3 empresas da região e nenhuma respondeu. Em uma delas, porém, eu tinha um interesse maior e resolvi dar as caras.

Então enviei um email mais ou menos assim:

> Olá.
>
> Há não muito tempo enviei meu currículo para vocês e não obtive resposta. Eu sei o que isso quer dizer e vim perguntar os motivos de eu não ter sido selecionado para que eu possa desenvolver minhas habilidades e meu currículo no rumo certo. Assim eu posso me tornar um bom desenvolvedor front end. Por favor, sejam sinceros. Eu aguento uma boa crítica.
>
> Obrigado!


Recebi um email no mesmo dia dizendo:

> Pedro, sobre o seu currículo, nós recebemos e analisamos ele, mas como você não se tinha algum vínculo com alguma faculdade desconsideramos pois a vaga é de estágio, mas devido ao seu interesse, você poderia dar um pulo aqui pra gente conversar, a gente tem um horário na segunda às 10h e às 16h.

No final das contas não preenchi a vaga, mas fui lá, fiz uma entrevista/conversa rápida com um dos sócios da agência (que é bem relacionado com outras agências aqui da região), me deu um direcionamento do que eu deveria estudar para ter mais chances na área.

Foram dicas muito úteis pra mim.

Ele elogiou particularmente a iniciativa de pedir um feedback e que foi só por isso que ele pediu que eu fosse até. Parece que quase ninguém havia feito isso antes em processos seletivos deles e isso chamou muita atenção, segundo ele.

Agora, no caso de você pedir uma resposta e te ignorarem, não acho que muito possa ser feito.

**Comment by brunoravanhani** [comment]
Isso já aconteceu comigo e sinceramente é uma grande falta de respeito com o candidato, pois geralmente é só um simples email e pronto, já deu feedback. 

Uma outra coisa que me chama atenção é que, pelo menos nas experiências que eu tive, sempre ultrapassam o prazo dado para o feedback.

Mas o que é pior e uma falta de respeito ainda maior, é quando você se dispõe a ir fazer entrevista, passa uma semana fazendo um teste, tentando conciliar com o seu tempo de trabalho e estudo e no fim não tem a coragem de dar uma simples resposta.

**Comment by viniciusdocanto** [comment]
Eu nunca dei muita expectativa cadastro de currículo, desde 2012 quando me mudei pra capital do RJ, mudei de emprego umas 4 vezes, fiz umas 5 entrevista de emprego, a unica que demorou pra responder, foi a primeira que demorou 2 meses pra dar um retorno, acho que tive muita sorte com as entrevistas.

**Comment by Shabaoth** [comment]
@MarcoWorms estao contratando?? rsrsr
